query/src/main/java/io/keen/client/java/Query.java

```diff
@@ -0,0 +1,416 @@
+         */
+        public ArrayList<String> getGroupBy() {return groupBy;}
+        /**
+         * Set group by
+         * @param groupBy the group by argument.
+         */
+        public void setGroupBy(ArrayList<String> groupBy) {
+            this.groupBy = groupBy;
+        }
+        /**
+         * Set GroupBy. This adds an additional GroupBy argument, and when called
+         * multiple times during method chaining, it can set a list of GroupBy's.
+         *
+         * @param groupBy the group by String.
+         * @return This instance (for method chaining).
+         */
+        public QueryBuilder withGroupBy(String groupBy) {
+            if (this.groupBy == null) {
+                this.groupBy = new ArrayList<String>();
+            }
+            this.groupBy.add(groupBy);
+            return this;
+        }
+        /**
+         * Set the GroupBy
+         * @param groupBy the ArrayList of properties to group by.
+         * @return This instance (for method chaining).
+         */
+        public QueryBuilder withGroupBy(ArrayList<String> groupBy) {
+            setGroupBy(groupBy);
+            return this;
+        }
+
+        /**
+         * get max age
+         * @return the max age.
+         */
+        public Integer getMaxAge() {return maxAge;}
+        /**
+         * Set max age
+         * @param maxAge the max age.
+         */
+        public void setMaxAge(Integer maxAge) {this.maxAge = maxAge;}
+        /**
+         * Set max age
+         * @param maxAge the max age.
+         * @return This instance (for method chaining).
+         */
+        public QueryBuilder withMaxAge(Integer maxAge) {
+            setMaxAge(maxAge);
+            return this;
+        }
+
+        /**
+         * get the percentile
+         * @return the percentile.
+         */
+        public Double getPercentile() {return percentile;}
+        /**
+         * Set percentile
+         * @param percentile the percentile.
+         */
+        public void setPercentile(Double percentile) {this.percentile = percentile;}
+        public void setPercentile(Integer percentile) {this.percentile = percentile.doubleValue();}
+        /**
+         * Set percentile
+         * @param percentile the percentile (type Double).
+         * @return This instance (for method chaining).
+         */
+        public QueryBuilder withPercentile(Double percentile) {
+            setPercentile(percentile);
+            return this;
+        }
+        /**
+         * Set percentile
+         * @param percentile the percentile (type Integer).
+         * @return This instance (for method chaining).
+         */
+        public QueryBuilder withPercentile(Integer percentile) {
+            setPercentile(percentile.doubleValue());
+            return this;
+        }
+
+        /**
+         * get latest
+         * @return the latest.
+         */
+        public Integer getLatest() {return latest;}
```

Comment:
Nit: Latest is used for extractions; since we pulled extractions from this initial release, maybe remove this as well so it doesn't confuse people?